features/clipboard-events.yml

```diff
@@ -1,7 +1,7 @@
-draft_date: 2024-08-28
-name: Clipboard API and events
-description: TODO
-spec: https://w3c.github.io/clipboard-apis/
+name: Clipboard Events
+description: The `ClipboardEvent` interface represents events providing information about system clipboard activity such as cut, copy, and paste.
+spec: https://w3c.github.io/clipboard-apis/#clipboard-event-interfaces
+group: clipboard
 compat_features:
   - api.ClipboardEvent
```

Comment:
This did not result in any change to the resulting dist file. Is it still needed?

Reply:
I'd added it anyway, so resolving. Just wanted to note that.

Reply:
Hmm, the reason it doesn't work is because `compute_from` is not in a status block, so it's not doing anything. That should have caused an error in `dist`—I'll look into why it didn't happen.

Reply:
Ah, it's because we don't actually check the shape of the YAML _before_ generating dists, only after we've created the final object for publishing. I've filed https://github.com/web-platform-dx/web-features/issues/1720

Reply:
AHA, yes would be great to catch invalid yaml in dist, thanks! fixed here